test/spectests/spectests.cpp

```diff
@@ -542,10 +539,14 @@ class test_runner
 
     bool check_result(fizzy::Value actual_value, const json& expected)
     {
+        // TODO: Check types here.
+        // TODO: Use read_value() here too if it can handle "nan:..." strings.
```

Comment:
Well `Value` can't represent expectation like "arithmetic NaN expected"

Reply:
True. But you can extend the Value to handle that or handle `nan:...` first using strings and then `read_value()`.